test/blackbox-tests/test-cases/include-qualified/group-module-overlap.t

```diff
@@ -0,0 +1,26 @@
+Dune should not allow mod.ml and mod/ in the same directory:
```

Comment:
these module overlapping tests should fail, but they're not. is that something you plan to fix later?

Reply:
Yes, once I make the call about where interface modules will go.